src/features/search/pages/SearchFilter.tsx

```diff
@@ -15,10 +15,20 @@ export const SearchFilter: React.FC = () => {
   const { searchState } = useSearch()
   const { data: profile } = useUserProfileInfo()
 
+  const scrollViewRef = useRef<ScrollView | null>(null)
+
+  useEffect(() => {
+    if (searchState.date && scrollViewRef !== null) scrollViewRef.current?.scrollToEnd()
+  }, [searchState.date])
+
+  useEffect(() => {
+    if (searchState.timeRange && scrollViewRef !== null) scrollViewRef.current?.scrollToEnd()
+  }, [searchState.timeRange])
+
   return (
     <React.Fragment>
       <React.Fragment>
-        <Container>
+        <Container ref={scrollViewRef as any}>
```

Comment:
```suggestion
        <Container ref={scrollViewRef}>
```
et tu fais `StyledScrollView = styled(ScrollView)(blabla)` pour ne pas avoir recours au `any`

Reply:
Vérifies quand même que ton style est bien appliqué

Reply:
je ne pense pas que ça fonctionne car justement le code que tu as écrit existe déjà Container = styled.ScrollView()